pawno/include/PPC_PlayerCommands.inc

```diff
@@ -1,27 +1,28 @@
 // This is just a joke command
 COMMAND:me(playerid, params[])
 {
-	// Setup local variables
-	new Msg[128], Message[MAX_STRING_INPUT_TEXT], Name[MAX_PLAYER_NAME];
```

Comment:
We should keep the declaration of all variables that will be used in the command in the first line. This gives a better overview in case we need to adjust them.